.github/workflows/build-hw-reusable.yml

```diff
@@ -7,13 +7,21 @@ on:
       adapter_name:
         required: true
         type: string
+      other_adapter_name:
+        required: false
+        type: string
+        default: ""
```

Comment:
Not sure if that would complicate things but it feels that if we make adapter name as a list of strings and use it might be more intuitive and expandable in the future than to limit it to only the main adapter and `other_adapter`.

Reply:
I think that would require UR changes to allow a list of adapters to be specified in a single cmake variable, unless yaml has any fancy string mapping abilities.

Reply:
Cool, I think lets keep it simple better then.